packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -18,6 +18,8 @@ import { captureException } from '@sentry/core';
 import { formGroups } from './formGroups';
 import { userOverNKarmaFunc } from "../../vulcan-users";
 
+const MINIMUM_COAUTHOR_KARMA = 10;
```

Comment:
After discussing with JP, we decided to lower this limit from 100 to 10 (it's not a typo!)